exercises/01-composite-ui/before/Divergent.Frontend/index.html

```diff
@@ -70,7 +70,6 @@
 
     <!-- Finance module -->
     <script src="/app/modules/finance/_module.js" type="text/javascript"></script>
-    <script src="/app/modules/finance/ordersLoadedSubscriber.js" type="text/javascript"></script>
```

Comment:
Why remove this line? Are you aware the exercise tries to add that file later on [over here](https://github.com/Particular/Workshop.Microservices/tree/master/exercises/01-composite-ui#step-2-1)?

Reply:
We could theoretically remove it, but then we'd need to update the exercise to add this line, otherwise the created javascript won't be loaded.
You can test in the "after" folder for this exercise. Remove it and the total for the order will never be retrieved.

Reply:
Sorry I wasn't aware its part of the exercise.
Usually I check for errors to see if the build passed successfully.
Thank you for your serious comment.

Reply:
We will make the changes tomorrow